data/users.js

```diff
@@ -1,20 +1,79 @@
-import {users} from '../config/mongoCollections.js';
-import {ObjectId} from 'mongodb';
-import validation from '../validation.js';
-
-let exportedMethods = {
-    async getAllUsers() {
-      const userCollection = await users();
-      const userList = await userCollection.find({}).toArray();
-      return userList;
-    },
-    async getUserById(id) {
-      id = validation.checkValidId(id,"user_id");
-      const userCollection = await users();
-      const user = await userCollection.findOne({_id: ObjectId(id)});
-      if (!user) throw 'Error: User not found';
-      return user;
-    }
-};
-
-export default exportedMethods;
```

Comment:
I just changed the way we export functions because to avoid exporting entire module of functions into other files where you might only need one fuction, but the previous version would import the entire module which is just unnecessary